modules/ztp-image-based-upgrade-creating-backup-resources-ztp.adoc

```diff
@@ -0,0 +1,228 @@
+// Module included in the following assemblies:
+// * scalability_and_performance/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-creating-backup-resources-with-ztp_{context}"]
+= Creating ConfigMap objects to back up resources with GitOps ZTP
```

Comment:
And I wonder if it would be clearer to move this whole section "Create Configmap objects with ZTP gitops" under "Upgrading single-node OpenShift clusters using Lifecycle Agent and gitops ztp "